jbdevstudio/multiple/jbdevstudio-multiple.target

```diff
@@ -294,6 +293,14 @@
     </location>
 
     <location includeAllPlatforms="false" includeMode="slicer" type="InstallableUnit" includeSource="true">
+      <repository location="http://download.jboss.org/jbosstools/updates/requirements/mylyn-extras/2.6.0.N20150706-2134/"/>
+      <!-- JBIDE-20216 wikitext asciidoc editor; exclude creole and commonmark plugins -->
+      <unit id="org.eclipse.mylyn.wikitext_feature.feature.group" version="2.6.0.N20150706-2134"/>
```

Comment:
Do we specifically need an alternative version for this feature? If not, I believe it would be better to stick to the version which is in Mylyn release or Simrel repositories.
@maxandersen Does your asciidoc extension require a newer version of the wikitext feature?